app/scripts/templates/ready.mustache

```diff
@@ -1,14 +1,6 @@
 <div id="main-content" class="card">
   <header>
-    {{#signUp}}
-    <h1 id="fxa-sign-up-complete-header">{{#t}}Account verified{{/t}}</h1>
-    {{/signUp}}
-    {{#resetPassword}}
-    <h1 id="fxa-reset-password-complete-header">{{#t}}Password reset{{/t}}</h1>
-    {{/resetPassword}}
-    {{#accountUnlock}}
-    <h1 id="fxa-account-unlock-complete-header">{{#t}}Account unlocked{{/t}}</h1>
-    {{/accountUnlock}}
+    {{{headerHTML}}}
```

Comment:
@shane-tomlinson generating that html header looks strange, can we change 
`{{{headerHTML}}}` to 

`<h1 id="{{headerId}}">{{headerTitle}}</h1>`